src/modules/quick_method_module.f90

```diff
@@ -555,11 +567,25 @@ subroutine read_quick_method(self,keywd)
             if (index(keywd,'SCF=') /= 0) self%iscf = rdinml(keywd,'SCF')
 
             ! DM Max RMS
-            if (index(keywd,'DENSERMS=') /= 0) self%pmaxrms = rdnml(keywd,'DENSERMS')
-
-            ! 2e-cutoff
+            if (index(keywd,'DENSERMS=') /= 0) then
+                self%pmaxrms = rdnml(keywd,'DENSERMS')
```

Comment:
@Altaimorin Chi, why can't we use read function in place of rdnml? Have you checked this?